lib/datadog/appsec/processor.rb

```diff
@@ -39,6 +39,27 @@ def finalize
         end
       end
 
+      class InvalidContextError < StandardError; end
+
+      class << self
+        def current_context
```

Comment:
WDYT of naming this `active_context`, in line with Tracing's `active_trace` and `active_span`?

Reply:
Sounds good 👍